frontend/lib/norent/letter-builder/tests/confirmation.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { AppTesterPal } from "../../../tests/app-tester-pal";
+import { NorentConfirmation, CANCEL_RENT_PETITION_URL } from "../confirmation";
+import { createProgressStepJSX } from "../../../progress/tests/progress-step-test-util";
+import { override } from "../../../tests/util";
+import { initNationalMetadataForTesting } from "./national-metadata-test-util";
+import { BlankOnboardingInfo } from "../../../queries/OnboardingInfo";
+
+beforeAll(initNationalMetadataForTesting);
+
+describe("NoRent confirmation page", () => {
+  const createPal = (state: string, trackingNumber?: string) => {
+    return new AppTesterPal(createProgressStepJSX(NorentConfirmation), {
+      session: {
+        phoneNumber: "1234567890",
+        onboardingInfo: override(BlankOnboardingInfo, {
+          state,
+        }),
+        norentLatestLetter: trackingNumber
+          ? {
+              trackingNumber: trackingNumber,
+              letterSentAt: null,
+              paymentDate: "Boop 1st, 2099",
+            }
+          : null,
+      },
+    });
+  };
+
+  it("should work", () => {
+    const pal = createPal("NY", "1234Boopy");
+    pal.rr.getByText("You've sent your letter");
+    expect(pal.rr.getByText("Sign the petition").getAttribute("href")).toBe(
+      CANCEL_RENT_PETITION_URL
+    );
```

Comment:
In #1429 I've added a helper function to `AppTesterPal` called `ensureLinkGoesTo` which should make this less verbose!

Reply:
Nice, just updated this.